pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -376,6 +378,17 @@ func (r *rbac) handleSCCInNamespace(ctx context.Context, ns *corev1.Namespace) e
 	return nil
 }
 
+func getRBACMaxCalls() int {
+	envValue := os.Getenv("OCP_RBAC_MAX_CONCURRENT_CALLS")
+	if envValue == "" {
+		return defaultRbacMaxConcurrentCalls
+	}
+	if parsedValue, err := strconv.Atoi(envValue); err == nil {
```

Comment:
@jkhelil if the error is not `nil`, print the error in the log and say, taking the default value: 20